app/assets/stylesheets/admin/loans/_colors.scss

```diff
@@ -0,0 +1,17 @@
+$green1: rgb(115,200,100);
```

Comment:
These are likely no longer needed if we are implementing colors from the backend.